tools/clang/test/HLSLFileCheck/passes/llvm/instcombine/nosink.hlsl

```diff
@@ -0,0 +1,38 @@
+// RUN: %dxc %s /T ps_6_0 -opt-enable sink | FileCheck %s
```

Comment:
I think it would be good to have a RUN line without an explicit `-opt-` flag for sink to show what the expected default is.

Reply:
Done.